src/main/java/it/pagopa/interop/probing/eservice/registry/updater/dao/EserviceDao.java

```diff
@@ -0,0 +1,163 @@
+/**************************************************************************
+*
+* Copyright 2023 (C) DXC
+*
+* Created on  : 7 mar 2023
+* Author      : dxc technology
+* Project Name: interop-be-probing-eservice-registry-updater 
+* Package     : it.pagopa.interop.probing.eservice.registry.updater.dao
+* File Name   : EserviceDao.java
+*
+*-----------------------------------------------------------------------------
+* Revision History (Release )
+*-----------------------------------------------------------------------------
+* VERSION     DESCRIPTION OF CHANGE
+*-----------------------------------------------------------------------------
+** --/1.0  |  Initial Create.
+**---------|------------------------------------------------------------------
+***************************************************************************/
+package it.pagopa.interop.probing.eservice.registry.updater.dao;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.UUID;
+
+import javax.persistence.EntityManager;
+import javax.persistence.Persistence;
+import javax.persistence.TypedQuery;
+
+import it.pagopa.interop.probing.eservice.registry.updater.config.PropertiesLoader;
+import it.pagopa.interop.probing.eservice.registry.updater.model.Eservice;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+/** The Constant log. */
+@Slf4j
+
+/**
+ * Gets the em.
+ *
+ * @return the em
+ */
+@Getter
+public class EserviceDao {
+
+	/** The instance. */
+	private static EserviceDao instance;
+
+	/** The Constant CONNECTION_PROP_URL. */
+	private static final String CONNECTION_PROP_URL = "javax.persistence.jdbc.url";
+
+	/** The Constant CONNECTION_PROP_USR. */
+	private static final String CONNECTION_PROP_USR = "javax.persistence.jdbc.user";
+
+	/** The Constant CONNECTION_PROP_PSW. */
+	private static final String CONNECTION_PROP_PSW = "javax.persistence.jdbc.password";
+
+	/** The Constant AURORA_URL. */
+	private static final String AURORA_URL = "amazon.aurora.url";
+
+	/** The Constant AURORA_USR. */
+	private static final String AURORA_USR = "amazon.aurora.user";
+
+	/** The Constant AURORA_PSW. */
+	private static final String AURORA_PSW = "amazon.aurora.password";
```

Comment:
I suggest to use `DB` instead of `AURORA`

Reply:
done